doc/manual/parallel-computing.rst

```diff
@@ -499,15 +499,17 @@ specified, it is called on all the participating workers.  You can
 arrange it so that each worker runs the ``init`` function on a
 distinct portion of the array, thereby parallelizing initialization.
 
-Here's a brief example::
+Here's a brief example:
+
+.. doctest::
 
   julia> addprocs(3)
-  3-element Array{Any,1}:
+  3-element Array{Int64,1}:
    2
    3
    4
 
-  julia> S = SharedArray(Int, (3,4), init = S -> S[localindexes(S)] = myid())
+  julia> S = SharedArray(Int, (3,4), init = S -> S[Base.localindexes(S)] = myid())
```

Comment:
Must've been unexported when DArrays were removed, judging by where it used to sit in the exports list: https://github.com/JuliaLang/julia/blob/6c7cbcedaf6264b466415de74901968911cca9e9/base/exports.jl#L1152